dev-haskell/hackage-security/hackage-security-0.6.0.1.ebuild

```diff
@@ -14,12 +14,9 @@ SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0/${PV}"
-# keep in sync with ghc-8.10
-# KEYWORDS="~amd64 ~x86"
+KEYWORDS="~amd64 ~x86"
```

Comment:
This cannot be keyworded unless `cabal-install-3.2.0.0` is also keyworded, which it does not yet appear to be in this PR.

Reply:
@solpeth can you elaborate on this? I don't see any dependencies for `cabal-install` in this ebuild. Indeed, I can confirm that it builds fine in my test environment with no `cabal-install` installed.

Reply:
This PR does not keyword `cabal-install-3.2.0.0`. I cannot find it in your `bump cabal-install` commit or anywhere else. I think you intended to keyword `cabal-install-3.2.0.0`. I think a previous iteration of this PR did do that. If I'm missing it, please point me in the right direction!

Consequentially, if there is no `cabal-install-3.2.0.0` then there is no reason to keyword `hackage-security-0.6.0.1` as you have done above. But I suspect that you did intend to keyword `cabal-install-3.2.0.0`.

Solution:

Actually keyword `cabal-install`. I believe it may have mistakenly been missed :)